docs: update cloud data sources README links (#37820)
grafana/grafana: public/app/plugins/datasource/cloud-monitoring/README.md

> # Google Cloud Monitoring Data Source - Native Plugin (formerly named Stackdriver)

Reviewer: ```suggestion
# Google Cloud Monitoring data source - native plugin (formerly named Stackdriver)
```

> Grafana ships with **built in** support for Google Cloud Monitoring. You just have to add it as a data source and you will be ready to build dashboards for your Cloud Monitoring metrics.

Reviewer 2: ```suggestion
Grafana ships with **built in** support for Google Cloud Monitoring. Simply add it as a data source and you are ready to build dashboards for your Cloud Monitoring metrics.
```
